worker/mutation.go

```diff
@@ -609,8 +609,8 @@ func AssignNsIdsOverNetwork(ctx context.Context, num *pb.Num) (*pb.AssignedIds,
 	return c.AssignIds(ctx, num)
 }
 
-// AssignUidsOverNetwork sends a request to assign UIDs to blank nodes to the current zero leader.
-func AssignUidsOverNetwork(ctx context.Context, num *pb.Num) (*pb.AssignedIds, error) {
+// AssignIDsOverNetwork sends a request to assign UIDs/TSs from the current zero leader.
+func AssignIDsOverNetwork(ctx context.Context, num *pb.Num) (*pb.AssignedIds, error) {
```

Comment:
Maybe you need to remove num.TYpe = pb.Num_UID from this function? Also remove the AssignNsIdsOverNetwork function?